### PR TITLE
Prep community.aws 9.0.0

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Prep community.aws 9.0.0

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by alinabuzachis** [comment]
recheck

File: galaxy.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+version: 9.0.0
 readme: README.md
 authors:
   - Ansible (https://github.com/ansible)
 description: A variety of Ansible content to help automate the management of AWS services.
 license_file: COPYING
 tags: [community, aws, cloud, amazon]
 dependencies:
-  amazon.aws: '>=8.0.0-dev0'
+  amazon.aws: '>=10.0.0-dev0'
```
Comment (GomathiselviS) [inline galaxy.yml:12]:
```suggestion
  amazon.aws: '>=9.0.0,<10.0.0'
```

**Review by GomathiselviS — APPROVED** [review]

**Comment by alinabuzachis** [comment]
recheck

**Review by markuman — APPROVED** [review]

**Comment by alinabuzachis** [comment]
regate

**Pull request merged**